provider/postgis/util_internal_test.go

```
@@ -123,23 +124,25 @@ func TestDecipherFields(t *testing.T) {
 		expectedTags     map[string]interface{}
 	}
 
-	cc := pgx.ConnConfig{
-		Host:     os.Getenv("PGHOST"),
-		Port:     5432,
-		Database: os.Getenv("PGDATABASE"),
-		User:     os.Getenv("PGUSER"),
-		Password: os.Getenv("PGPASSWORD"),
-	}
+	host := os.Getenv("PGHOST")
+	port := 5432
+	db := os.Getenv("PGDATABASE")
+	user := os.Getenv("PGUSER")
+	password := os.Getenv("PGPASSWORD")
+
+	cs := fmt.Sprintf("postgres://%v:%v@%v:%v/%v", user, password, host, port, db)
+	dbconfig, _ := BuildDBConfig(cs)
```

any specific reason to ignore the error here?